app/src/main/java/io/left/meshim/controllers/RightMeshController.java

```diff
@@ -388,5 +405,41 @@ void handleDataDelivery(RightMeshEvent e) {
             unDeliveredMessageIds.remove(deliveryDataId);
             updateInterface();
         }
+        else if(undeliveredPeerUpdateMessages.containsKey(deliveryDataId)){
```

Comment:
This should be on the same line, maybe run Checkstyle to make sure there isn't something I'm missing.

Reply:
nope! it was my bad